libs/dscache/odc/dscache/apps/dstiler.py

```diff
@@ -36,6 +40,17 @@ def cli(native, native_albers, web, dbfile):
         gs = web_gs(web)
         group_key_fmt = 'web_' + str(web) + '/{:03d}_{:03d}'
         binner = lambda dss: bin_dataset_stream(gs, dss)
+    elif crs is not None or resolution is not None or shape is not None or bin_format is not None:
```

Comment:
this condition looks odd, but I guess it's right, I guess `any(x is not None for x in [crs, resolution, shape, bin_format])` is not that much better